From a9211094d967c933c51cd31513ea189b5881646f Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Thu, 31 Jul 2014 16:03:39 -0500
Subject: [CHANGE 17/31] qemu-iotests: Disable cases requiring not backported
 features
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1406822631-6570-18-git-send-email-kwolf@redhat.com>
Patchwork-id: 60368
O-Subject: [RHEL-6.6 qemu-kvm PATCH v3 17/29] qemu-iotests: Disable cases requiring not backported features
Bugzilla: 1122410
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Max Reitz <mreitz@redhat.com>

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
Signed-off-by: jen <jen@redhat.com>
---
 tests/qemu-iotests/group | 50 ++++++++++++++++++++++++------------------------
 1 file changed, 25 insertions(+), 25 deletions(-)

diff --git a/tests/qemu-iotests/group b/tests/qemu-iotests/group
index 13c5ceb..bf67a07 100644
--- a/tests/qemu-iotests/group
+++ b/tests/qemu-iotests/group
@@ -42,61 +42,61 @@
 033 rw auto quick
 034 rw auto backing quick
 035 rw auto quick
-036 rw auto quick
+# 036 rw auto quick -- requires qcow2 v3 (feature bits)
 037 rw auto backing quick
 038 rw auto backing quick
-039 rw auto quick
+# 039 rw auto quick -- required qcow2 v3 (lazy refcounts)
 040 rw auto
 041 rw auto backing
 042 rw auto quick
 043 rw auto backing
 044 rw auto
-045 rw auto quick
-046 rw auto aio quick
-047 rw auto quick
+# 045 rw auto quick -- requires add-fd
+# 046 rw auto aio quick -- requires blkdebug break/resume
+# 047 rw auto quick -- requires discard in qemu-io
 048 img auto quick
 049 rw auto
-050 rw auto backing quick
-051 rw auto
+# 050 rw auto backing quick -- requires qcow2 v3 (zero clusters)
+# 051 rw auto -- requires driver-specific options
 052 rw auto backing quick
-053 rw auto quick
+# 053 rw auto quick -- compressed images are unsupported
 054 rw auto quick
-055 rw auto
-056 rw auto backing
-057 rw auto
-058 rw auto quick
+# 055 rw auto -- requires drive-backup
+# 056 rw auto backing -- requires drive-backup
+# 057 rw auto -- requires QMP support for internal snapshots
+# 058 rw auto quick -- requires qemu-nbd support for internal snapshots
 # 059 rw auto quick -- requires sample images
 060 rw auto quick
-061 rw auto quick
-062 rw auto quick
-063 rw auto quick
+# 061 rw auto quick -- requires qemu-img amend
+# 062 rw auto quick -- requires qcow2 v3 (zero clusters)
+# 063 rw auto quick -- requires qemu-img convert -n
 # 064 rw auto quick -- requires sample images
-065 rw auto quick
-066 rw auto quick
+# 065 rw auto quick -- requires driver-specific qemu-img info fields
+# 066 rw auto quick -- requires qcow2 v3 (zero clusters)
 067 rw auto quick
 068 rw auto quick
 069 rw auto quick
 # 070 rw auto quick -- requires sample images
-071 rw auto quick
-072 rw auto quick
+# 071 rw auto quick -- requires driver-specific options in blkdebug/blkverify
+# 072 rw auto quick -- requires blockdev (nested image formats)
 073 rw auto quick
 074 rw auto quick
 # 075 rw auto quick -- requires sample images
 # 076 auto -- requires sample images
-077 rw auto quick
+# 077 rw auto quick -- requires blkdebug break/resume and 4k sector support
 # 078 rw auto quick -- requires sample images
 079 rw auto
 080 rw auto
 081 rw auto quick
-082 rw auto quick
+# 082 rw auto quick -- requires qemu-img support for multiple -o
 083 rw auto
 084 img auto quick
 085 rw auto
 086 rw auto quick
-087 rw auto quick
+# 087 rw auto quick -- requires blockdev-add
 088 rw auto quick
-089 rw auto quick
-090 rw auto quick
-091 rw auto quick
+# 089 rw auto quick -- requires json: pseudo-protocol
+# 090 rw auto quick -- requires discard in qemu-io
+# 091 rw auto quick -- required 'qemu-io' HMP command
 092 rw auto quick
 095 rw auto quick
-- 
1.9.3

